src/camera_line_follower_component.cpp

```diff
@@ -41,64 +42,59 @@ Camera_Follower::Camera_Follower(const rclcpp::NodeOptions & options)
 
 void Camera_Follower::image_callback(const sensor_msgs::msg::Image::SharedPtr msg_image)
 {
-  auto cv_img = cv_bridge::toCvShare(msg_image, msg_image->encoding);
-  auto msg = std::make_unique<sensor_msgs::msg::Image>();
+  const auto cv_img = cv_bridge::toCvShare(msg_image, msg_image->encoding);
   auto result_msg = std::make_unique<sensor_msgs::msg::Image>();
-  msg->is_bigendian = false;
-  result_msg->is_bigendian = false;
 
   cv::Mat frame, result_frame;
   cv::cvtColor(cv_img->image, frame, CV_RGB2BGR);
 
   if (!frame.empty()) {
-    following(frame, result_frame);
+    detecting_line(frame, result_frame);
     convert_frame_to_message(result_frame, *result_msg);
     result_image_pub_->publish(std::move(result_msg));
   }
 }
 
 void Camera_Follower::callback_switches(const raspimouse_msgs::msg::Switches::SharedPtr msg)
 {
-  switches_ = *msg;
+  if (msg->switch0) {
+    RCLCPP_INFO(this->get_logger(), "Stop following.");
+    enable_following_ = false;
+  } else if (msg->switch2) {
+    RCLCPP_INFO(this->get_logger(), "Start following.");
+    enable_following_ = true;
+  }
 }
 
 void Camera_Follower::on_cmd_vel_timer()
 {
-  const double LINEAR_VEL = 0.2;  // unit: m/s
-  const double ANGULAR_VEL = -0.8;  // unit: rad/s
-  const double TARGET_AREA = 0.1;  // 0.0 ~ 1.0
-  const double OBJECT_AREA_THRESHOLD = 0.01;  // 0.0 ~ 1.0
+  constexpr double LINEAR_VEL = 0.2;  // unit: m/s
+  constexpr double ANGULAR_VEL = -0.8;  // unit: rad/s
+  constexpr double TARGET_AREA = 0.1;  // 0.0 ~ 1.0
+  constexpr double OBJECT_AREA_THRESHOLD = 0.01;  // 0.0 ~ 1.0
+  geometry_msgs::msg::Twist cmd_vel;
 
-  // Detects an object and tracks it
+  // Detect and follow the line
   // when the number of pixels of the object is greater than the threshold.
```

Comment:
ここではDetectしてないので訂正してほしいです。（画像処理もここで行われている　と勘違いします）

```suggestion
  // Follow the line
  // when the number of pixels of the object is greater than the threshold.
```